nomenclature/codelist.py

```diff
@@ -27,7 +27,7 @@ def read_validation_schema(i):
     return schema
 
 
-SCHEMA_TYPES = ("variable", "tag", "region", "generic")
+SCHEMA_TYPES = ("variable", "tag", "region", "generic", "meta")
```

Comment:
In conjunction with my comment about the meta schema I would leave it out here and just go with generic for now.
```suggestion
SCHEMA_TYPES = ("variable", "tag", "region", "generic")
```
I'll most likely remove all the schemas anyway in an upcoming PR.